src/main/kotlin/com/quipalup/katydid/logentry/primaryadapter/httprest/CreateLogEntryEndpoint.kt

```diff
@@ -0,0 +1,31 @@
+package com.quipalup.katydid.logentry.primaryadapter.httprest
+
+import com.quipalup.katydid.common.domain.Id
+import com.quipalup.katydid.logentry.domain.LogEntry
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+class CreateLogEntryEndpoint {
+    @PostMapping("/log-entries")
+    @ResponseStatus(HttpStatus.CREATED)
+    fun execute(@RequestBody newLog: LogEntry): MutableList<LogEntryResource> {
```

Comment:
```suggestion
    fun execute(@RequestBody newLog: LogEntryRequestDocument): LogEntryResponseDocument {
```

When you resolve the comment above, you'll see that the framework get the request json and it can't to instantiate an object LogEntry from it. This is due to that the json (that is compliant with JSON:API) we receive doesn't have the same data structure of our domain class `LogEntry`. I mean fields like "data", "type", "attributes". It is a good practice to keep your domain code independent from HTTP data details, so the best way to achieve this is to create another data structure to represent request and response JSON's and keep domain classes as they are. We'll see later how to transform HTTP data into domain data. In other comment you'll see the definition of this new classes.